pkg/nvml/lib.go

```diff
@@ -85,6 +113,10 @@ func (l *library) load() (rerr error) {
 		return fmt.Errorf("error opening %s: %w", l.path, err)
 	}
 
+	// Update the errorStringFunc to point to nvml.ErrorString
+	errorStringFunc = nvmlErrorString
```

Comment:
Do we need to set this to the default once we close the library?

Reply:
Yes. Fixed.